config/config.go

```diff
@@ -1285,3 +1287,17 @@ type TmaxAdjustments struct {
 	// PBS won't send a request to the bidder if the bidder tmax calculated is less than the BidderResponseDurationMin value
 	BidderResponseDurationMin uint `mapstructure:"bidder_response_duration_min_ms"`
 }
+
+type Auction struct {
+	PrivacySandbox PrivacySandbox `mapstructure:"privacysandbox"`
+}
+
+type PrivacySandbox struct {
+	TopicsDomain      string            `mapstructure:"topicsdomain"`
```

Comment:
`TopicsDomain` is for a separate issue. Please remove from this PR.

Reply:
done